### DISCUSSION TITLE
Does not show data with recipient profile

### DISCUSSION THREAD

**Discussion by swoopito** [globaleaks/globaleaks-whistleblowing-software#3946] [Q&A]
Hello everyone.

I have globaleaks installed on a server with debian, but when I log in with a recipient profile it does not show the logs entered.

![Destinatario](https://github.com/globaleaks/GlobaLeaks/assets/156353855/ebec02c5-2510-4936-8738-fb9cb2cd5379)

When I enter with an analyst profile it shows me the following.

![Analista](https://github.com/globaleaks/GlobaLeaks/assets/156353855/b6b25546-d924-45e7-9c4c-15168c306c06)

And with the administrator there is no problem.

If anyone has any information I would greatly appreciate it.

Greetings.

**Comment by evilaliv3** [comment]
Hello, 

i suspect you may have created multiple recipients but you may have forgot to attach some of them to the reporting channels.
